scripts/build-jitsi.js

```diff
@@ -0,0 +1,22 @@
+// This is a JS script so that the directory is created in-process on Windows.
```

Comment:
Can you expand this to elaborate why it's need on Windows? Did the shell script version allow mkdir to complete before the directory actually exists...?

Reply:
There's a risk of that, though the more concerning problem is that unless the Windows user has set up all their file associations correctly (unlike me, apparently) the script just won't run.